modules/renter/renter_test.go

```diff
@@ -27,6 +27,7 @@ type renterTester struct {
 	walletKey crypto.TwofishKey
 
 	renter *Renter
+	dir    string
```

Comment:
Do we need this now that we moved the test to the siatest package?

Reply:
all the other modules have it